UnityProject/Assets/Scripts/Systems/Research/LaserPuzzle/ResearchLaserProjector.cs

```diff
@@ -172,22 +179,28 @@ public void ServerPerformInteraction(HandApply interaction)
 			return;
 		}
 
-		if (Emitter.ValidSetup() == false)
+		if (Emitter.ValidSetup(true) == false)
 		{
 			Chat.AddActionMsgToChat(this.gameObject, " The research laser beeps and boops Stating Still needs to be constructed fully and Powered");
 			return;
 
 		}
 
-		if (interaction.IsAltClick)
-		{
-			FireLaser();
-		}
-		else
+
+		if (interaction.IsAltClick == false)
 		{
-			Chat.AddActionMsgToChat(this.gameObject, " The research laser beeps and boops. Firing test Projection, Remember to connect collectors to research laser ");
+			if (Emitter.IsOn)
+			{
+				FireLaser();
+			}
+			else
+			{
+				Chat.AddActionMsgToChat(this.gameObject, " The research laser beeps and boops. Firing test Projection, Remember to connect collectors to research laser ");
```

Comment:
```suggestion
				Chat.AddActionMsgToChat(this.gameObject, "The research laser beeps and boops. Firing test projection. Remember to connect collectors to research laser ");
```

Reply:
"the" 
isn't needed since it can do multiple so it isn't a singular projection, 
it's from the point of view of the machine so e.g, 

firing 9mm bullet vs firing the 9mm bullet

Reply:
alright